src/sinks/clickhouse/config.rs

```diff
@@ -33,10 +33,16 @@ pub struct ClickhouseConfig {
     /// The database that contains the table that data will be inserted into.
     #[configurable(metadata(docs::examples = "mydatabase"))]
     pub database: Option<String>,
+    /// If true`, ClickHouse Native Protocol is used. Defaults to `false`, using `JSONEachRow` over HTTP.
```

Comment:
These need `configurable` annotations. You can see the other fields here for examples.